### PR TITLE
Add pyprland

### LINKED ISSUE #32: Pyprland isn't listed in plugins !!!
And that's really a shame ;)

### PR DESCRIPTION
**Pull request by yavko**
closes #32

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ Awesome list for Hyprland, that includes useful tools and libraries that either
 - [hyprtrails](https://github.com/hyprwm/hyprland-plugins/tree/main/hyprtrails) ![c++][cpp] (Adds trails behind windows)
 - [cs:go vulkan fix](https://github.com/hyprwm/hyprland-plugins/tree/main/csgo-vulkan-fix) ![c++][cpp] (Fixes custom resolutions on CS:GO with -vulkan)
 
+### IPC plugins
+These technically aren't hyprland plugins, but extend hyprland functionality using only IPC
+- [pyprland](https://github.com/hyprland-community/pyprland) ![python][py] (An IPC plugin framework, with many built-in plugins)
```
Comment (fdev31) [inline README.md:47]: Maybe worth mentioning the scratchpads since it's the most popular plugin... but it's also clear this way.

**Review by fdev31 — COMMENTED** [review]
Looks good to me.

File: README.md (hunk 1 above)
Comment (BackHDLP) [reply README.md:47]: Maybe use nested lists for pyprland plugins?
Something like below could work, I think

```
- [pyprland](https://github.com/hyprland-community/pyprland) ![python](py) 
  - plugin using pyprland
  - another plugin using pyprland
```

Comment (yavko) [reply README.md:47]: good idea actually

**Comment by BackHDLP** [comment]
Any reason this or #46 isn't merged yet?

**Comment by yavko** [comment]
New pr exists

**Pull request closed without merging**